scripts/three-sixty.js

```diff
@@ -236,20 +250,26 @@ H5P.ThreeSixty = (function (EventDispatcher, THREE) {
     /**
      * Add element to "CSS 3d world"
      *
-     * @param {DOMElement} element
+     * @param {HTMLElement} element
      * @param {Object} startPosition
      * @param {boolean} enableControls
      */
     self.add = function (element, startPosition, enableControls) {
-      var threeElement = new THREE.CSS2DObject(element);
+      let threeElement;
+      if(element.classList.contains("render-in-3d")){
+        threeElement = new THREE.CSS3DObject(element);
+        threeElement.is3d = true;
+      } else {
```

Comment:
Please follow the same `if` `else` block style already established in the file.
Note that `if` is not a function call and should have whitespace unless minified.